docs/cita/rpc-guide/rpc.md

```diff
@@ -66,6 +68,50 @@ Result:
 
 ***
 
+### peersInfo
+
+获取与本节点相连的其它节点信息，信息包括节点的 `address` 和节点 `ip`。
+
+* Parameters
+
+None
+
+* Returns
+
+Object - A peersInfo object:
+
+* amount: `Quantity` - The amount of peers connected to this node.
+* peers: `Object` - peers info, including peer address and peer ip.
+* errorMessage: `String` - execution error message.
```

Comment:
`peersInfo` object doesn't contain  `errorMessage`

Reply:
But it contains, you can try:

```
curl -X POST --data '{"jsonrpc":"2.0","method":"peersInfo","params":[],"id":83}'
```

it will return:

```
{
    "jsonrpc":"2.0",
    "id":83,
    "result":{
        "amount":3,
        "peers":{
            "0x3ea53608732da3761ef41805da73f0d45d3e8e09":"127.0.0.1",
            "0x01cb0a8012b75ea156eaef3e827547f760dd917a":"127.0.0.1",
            "0x2aaeacf658e49f58973b4ef6f37a5c574a28822c":"127.0.0.1"
        },
        "errorMessage":null
    }
}
```

Reply:
OK. Forget it.